devel/ci/bodhi-ci

```diff
@@ -458,8 +458,12 @@ class Job(object):
                     self._pre_start_hook()
                     self._start_time = datetime.datetime.utcnow()
                     self.started = True
-                    process = await asyncio.create_subprocess_exec(*self._command,
-                                                                   **self._popen_kwargs)
+                    if not self._popen_kwargs["shell"]:
+                        process = await asyncio.create_subprocess_exec(
+                            *self._command, **self._popen_kwargs)
+                    else:
+                        process = await asyncio.create_subprocess_shell(
+                            *self._command, **self._popen_kwargs)
```

Comment:
Ha, at first I thought the two sides of this branch were the same. Now I see the exec/shell, and remember you mentioning this yesterday.